BossMod/Modules/RealmReborn/Trial/T09WhorleaterH/T09WhorleaterHSpinningDive.cs

```diff
@@ -5,52 +5,52 @@
 namespace BossMod.Modules.RealmReborn.Trial.T09WhorleaterH;
    class SpinningDive : GenericAOEs //TODO: Find out how to detect spinning dives earlier eg. the water column telegraph
     {     
-        private float activeHelper;       
+        private bool activeHelper;       
         private static AOEShapeRect rect = new(46, 8);
         public override IEnumerable<AOEInstance> ActiveAOEs(BossModule module, int slot, Actor actor)
         {
             var helper = module.Enemies(OID.SpinningDiveHelper).FirstOrDefault();      
-            if (helper != null && activeHelper >=1)
+            if (helper != null && activeHelper == true)
             yield return new(rect, helper.Position, helper.Rotation, new());
             }
         public override void OnActorCreated(BossModule module, Actor actor)
         {
             if ((OID)actor.OID == OID.SpinningDiveHelper)
-             activeHelper = 1;
+             activeHelper = true;
         }
         public override void OnEventCast(BossModule module, Actor caster, ActorCastEvent spell)
         {
             base.OnEventCast(module, caster, spell);
             if ((AID)spell.Action.ID == AID.SpinningDiveSnapshot)
             {
-                activeHelper = 0 ;
+                activeHelper = false;
             }
         }
     }
 
         class SpinningDiveKB : Knockback //TODO: Find out how to detect spinning dives earlier eg. the water column telegraph
     {        
-        private float activeHelper;       
+        private bool activeHelper;       
          public SpinningDiveKB() : base(ActionID.MakeSpell(AID.SpinningDiveEffect)) { }
         private static AOEShapeRect rect = new(46, 8);
         public override IEnumerable<Source> Sources(BossModule module, int slot, Actor actor)
         {
                 var helper = module.Enemies(OID.SpinningDiveHelper).FirstOrDefault();  
-                if (helper != null && activeHelper >=1)
+                if (helper != null && activeHelper == true)
```

Comment:
Minor thing, but `if (mybool == true)` is redundant, you can just use `if (mybool)`